app/MMB/src/scenes/timeline/MainContentNote.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import * as React from 'react';
+import { Text, Color, StyleSheet } from '@kiwicom/mobile-shared';
+import { type TranslationType } from '@kiwicom/mobile-localization';
+
+type Props = {|
+  +children: ?TranslationType,
+|};
+
+const Note = (props: Props): ?React.Node => {
```

Comment:
React.Node includes null, so you don't need the ?React.Node

Reply:
Ok, I will remove it.

Reply:
Done